Sources/Core/AWSClientRuntime/Auth/AWSCredentialIdentityResolvers/CachedAWSCredentialIdentityResolver.swift

```diff
@@ -5,13 +5,14 @@
 // SPDX-License-Identifier: Apache-2.0
 //
```

Comment:
`AWSCredentialIdentity` (above) was moved out to `smithy-swift` because SRA specifies that it should be there.